### DISCUSSION TITLE
[Feature Request]Add balance control

### DISCUSSION THREAD

**Discussion by audunn** [File-New-Project/EarTrumpet#644] [Ideas]
Add controls that allow you to control balance as well as volume.
Some times external factors limit speaker placement, in some cases this means you need to adjust balance depending on listening position. Balance controls in windows are buried  quite deep in audio settings and are a pain to readjust when windows resets the balance. This can happen after restarts etc.
Adding a convenient balance control easily accessible with the volume controls would be beneficial in those and similar situations.

**Comment by S0und** [comment]
I would give an other use case. If we could set the volume balance per app base, you could listen to two audio source at the same time, without interfering with each other on the same channel. 

So the application A could be:

- L - 25%
- R - 0%

application B:

- L - 0%
- R - 100%

You could focus on the youtube video, but still not deattached from listening to music or a stream.

**Reply by vertigo220** [reply]
Been using EarTrumpet for a while now and had a use for this today when on hold for a long time. Wanted to be able to have the phone call quiet on one side so it wouldn't be distracting but could hear when they finally answered and watch a movie on the other side while waiting, so of course was disappointed that I couldn't do it. Even worse, after >20 minutes on hold, I tried Voicemeeter to see if I could do it with that and it completely muted all audio and I couldn't get it back, and of course they answered a couple minutes later, so I had to do it all again. So having something like this built in and easy to use in ET would be great.

**Comment by ajmilner85** [comment]
Have been scouring the internet tonight looking for a way to do this (volume balance on a per application basis). Can't believe there's nothing available. Hopefully it's a feature that can be added to Ear Trumpet in the future!

**Comment by khag7** [comment]
This would be nice. Not a programmer, can't contribute, but if it gets done I'd gladly donate.

**Comment by un1r8okq** [comment]
Yes! I would love to have a tool that does this. I am a software engineer, possibly able to contribute

**Comment by JohnLoveDA** [comment]
This is not a question of possibility, it's absolutely possible.  There are audio libraries out there that already achieve this, just not in the context of what EarTrumpet offers.

**Comment by Kryptortio** [comment]
If this is implemented it might also be worth a look at implementing not only LR but also 3D ([binaural audio](https://www.youtube.com/watch?v=IUDTlvagjJA)/[3D sound](https://en.wikipedia.org/wiki/3D_sound_synthesis)). It's great being able to adjust the audio source like that so that you can easily focus on many things at once (it's like being able to put a virtual radio in the corner of your room). The LR feature could also be useful for hearing impaired.

**Comment by WolfganP** [comment]
100% supporting this. Adding balance control to individual apps will be a wonderful addition to Ear Trumpet.

**Comment by un1r8okq** [comment]
Is there anything we can do to get this prioritised? I think it would be a killer feature and I'd use it every day.

**Comment by thomasmockridge** [comment]
Great app but would really like to be able to set balance per app good examples above, for dealing with a slow support call while listening to music. Could be multiple destinations ie 5.1 but really lhs and rhs for each monitor ie LHS speaker for LHS monitor and RHS speaker for RHS monitor would be great.
+1 for this feature.

**Comment by Bobswesternshirt** [comment]
System-wide balance control is a must for those of us with limited hearing in one ear. Would love this feature!
